jest.config.js

```diff
@@ -1,4 +1,10 @@
 module.exports = {
-	collectCoverageFrom: ['src/**/*.js'],
-	testMatch: ['**/*.spec.js'],
+  collectCoverageFrom: ['src/**/*.{js,jsx}'],
+  testEnvironment: 'jsdom',
+  moduleDirectories: ['<rootDir>/src', 'node_modules'],
+  modulePaths: ['<rootDir>/src/'],
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
+  testMatch: ['**/*.spec.(tsx)', '**/*.spec.(ts|js)'],
+  transform: { '^.+\\.(ts|tsx|jsx|js)$': 'babel-jest' },
+  transformIgnorePatterns: ['/node_modules/(?!molekule).+\\.js$'],
```

Comment:
i don't see any remaining js source or test files, can we only match on ts/tsx now?